frontend/src/libs/components/prompt-generation/prompt-generation.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button, Icon } from '~/libs/components/components.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import {
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from '~/libs/hooks/hooks.js';
+
+import { PromptCard } from './libs/components/components.js';
+import { PromptCategory } from './libs/enums/enums.js';
+import { categoryPrompts } from './libs/mocks/mocks.js';
+import styles from './styles.module.scss';
+
+const { CHARACTER, SITUATION, SETTING, PROP, GENRE } = PromptCategory;
+
+const defaultPrompt = {
+  [CHARACTER]: 'Girl',
+  [SITUATION]: 'Street',
+  [SETTING]: '',
+  [PROP]: 'Red hair',
+  [GENRE]: 'Fiction',
+};
```

Comment:
Don't destruct enum - as enum is some sort of grouped values (in this case keys), then destructuring makes it harder to understand what entity is it
```suggestion
const defaultPrompt = {
  [PromptCategory.CHARACTER]: 'Girl',
  [PromptCategory.SITUATION]: 'Street',
  [PromptCategory.SETTING]: '',
  [PromptCategory.PROP]: 'Red hair',
  [PromptCategory.GENRE]: 'Fiction',
};
```